lib/logError.js

```diff
@@ -16,7 +16,6 @@ module.exports = function(error) {
     });
   } else {
     if (error.original) { logger.debug(error.original); }
-    logger.error();
```

Comment:
Figured as much... I'll add it back and think about if we can find a way to make this conditional without making things more complex.